### ISSUE TITLE
Denial Of Service when using HTTPS

### ISSUE THREAD

**Issue by pitust** [oakserver/oak#147]
# Targets
Any TLS Server using oak. This one will do:
```typescript
import { Application } from "https://deno.land/x/oak/mod.ts";

const app = new Application();

app.use((ctx) => {
    ctx.response.body = "Hello World!";
});

await app.listen({
    port: 8000,
    secure: true,
    certFile: 'x.cer',
    keyFile: 'x.pem'
});
```
# Exploit 
```typescript
fetch('http://localhost:8000');
```
This works *even* in the browser
# Expected behaviour
The server keeps working.
# Actual behaviour
The server crashes.

**Comment by pitust** [comment]
this _may_ be a bug in deno core. I am not sure

**Comment by kitsonk** [comment]
Yes, it is a Deno std library issue, which oak cannot fix: denoland/deno#5260

**Comment by kitsonk** [comment]
The upstream issue is resolved now.
